doc/github_actions/security_issues.rst

```diff
@@ -0,0 +1,102 @@
+security-issues
+===============
+
+Example Usage
+-------------
+
+.. code-block:: yaml
+
+    name: Report Security Issues for Repository
+
+    on:
+      schedule:
+        # “Every day at 00:00.” (https://crontab.guru)
+        - cron: "0 0 * * *"
+
+    jobs:
+
+      report_security_issues:
+
+        name: Report Security Issues
+
+        runs-on: ubuntu-latest
+
+        permissions:
+          issues: write
+
+        steps:
+          - name: SCM Checkout
+            uses: actions/checkout@v2
+
+          - name: Report Security Issues
+            uses: exasol/python-toolbox/.github/actions/security-issues@0.6.0/security-issues-action
+            with:
+              format: "maven"
+              command: "cat maven-cve-report.json"
+              github-token: ${{ secrets.GITHUB_TOKEN }}
+
+Configuration
+-------------
+This action exposes 3 configuration parameters `command`_, `format`_ and `github-token`_, for details see
+the specific sections below.
+
+command
++++++++
+
+Workspace command which shall be executed in order to check the project's dependencies for CVEs.
+
+.. note::
+
+    The calling workflow need to make sure the specified command can be executed in the context of the workflow.
+
+
+format
+++++++
+
+Specifies converter which needs to be applied on the output of the provided command.
+Currently there are only two converters available
+
+#. maven
+
+    Converts the output of mavens oss plugin into required input format.
+
+
+#. pass-through
+
+    In case the command itself already outputs the expected input format, the format can be specified as code:`pass-through`.
+
+
+Input Format
+------------
+
+The expect intput format is jsonl (line based json), of the following form:
+
+.. code-block:: python
+
+    { "cve": "<cve-id>", "cwe": "<cwe-id>", "description": "<multiline string>", "coordinates": "<string>", "references": ["<url>", "<url>", ...] }
```

Comment:
what does coordinates mean?

Reply:
Well, this is a term from the maven plugin/output, **TL;DR** it names the culprit (dependency + version)

Reply:
If you wan't to see how a generated ticket looks like see [here](https://github.com/Nicoretti/github-actions-playground/issues/29)